semantic_conventions.md

```diff
@@ -25,13 +25,15 @@ Span tags apply to **the entire Span**; as such, they apply to the entire timera
 | `http.method` | string | HTTP method of the request for the associated Span. E.g., `"GET"`, `"POST"` |
 | `http.status_code` | integer | HTTP response status code for the associated Span. E.g., 200, 503, 404 |
 | `http.url` | string | URL of the request being handled in this segment of the trace, in standard URI format. E.g., `"https://domain.net/path/to?resource=here"` |
+| `mom.destination` | string | An address at which messages can be exchanged. E.g. A Kafka record has an associated `"topic name"` that can be extracted by the instrumented producer or consumer and stored using this tag. |
+| `mom.type` | string | Message destination type, e.g. `"topic"` or `"queue"`. |
```

Comment:
I did not like the `db.type=sql/nosql` ,  `db.type` related of `db.statement`. If tracer want to analysis `statement`, `db.type=nosql` is not helpful for anything. 

By using `component` tag, I prefer the value is what framework component the application used. e.g. for redis in Java, component = Jedis